### DISCUSSION TITLE
Force installed extensions work in locked mode on Google forms

### DISCUSSION THREAD

**Discussion by Inglan2** [3kh0/ext-remover#1026] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
not rly a fix but there is not better category

Our school force installed ublock...

bad idea

because its force installed it works in Google forms locked mode so I can use something like uRun or ublock script executer to run script on Google forms.

I will make an exploit or something in spare time

**Comment by AshtonDavies** [comment]
What can you do with scripts in locked mode?

**Reply by Inglan2** [reply]
you can create a frame to Google or something so you can Google the answer or you could use some sort of generative ai like [librechat](https://librechat-librechat.hf.space/) if you can embed it and have an openai API key. I can write a script later today to work with ublock

**Reply by AshtonDavies** [reply]
Nice.

If this is useful to you, I would like to inform you that SpaceSaver2000 on Discord has made locked form hacking possible without bookmarklets. The launcher requires just the source data of the start quiz screen and then it opens it as a regular tab.

Instructions are [here](https://discord.com/channels/419123358698045453/1164377245956198430), or just the launcher link is [here](https://formb.pegoku.com).

**Reply by Blobby-Boi** [reply]
my school force installed ublock too loll

**Reply by Inglan2** [reply]
> If this is useful to you, I would like to inform you that SpaceSaver2000 on Discord has made locked form hacking possible without bookmarklets. The launcher requires just the source data of the start quiz screen and then it opens it as a regular tab.

If developer tools availability is 2 I think it will block view source but you can just add view-source: to the start of the url

**Reply by ohiobombs** [reply]
this works but it resets to a blank page whenever you leave the tab, and you can go in but your teacher will get multiple notifications of you signing in and out, which is weird during a locked mode quiz

**Reply by AshtonDavies** [reply]
I do know of another way to get answers for locked mode quizzes. If you study, I am pretty sure you'll get questions right.

**Reply by ohiobombs** [reply]
explained to the teacher and still got a 10/10 😭

**Comment by xNasuni** [comment]
you can use my script [xNasuni/google-forms-unlocker](https://github.com/xnasuni/google-forms-unlocker/) in part with the ublock script executor in place of the developer console if you wanna do your quizzes/tests without being in locked mode !! :3

**Reply by S-PScripts** [reply]
ok?
